lib/lotus/view/rendering/layout_registry.rb

```diff
@@ -9,7 +10,7 @@ module Rendering
       # This is raised at the runtime when Lotus::Layout cannot find it's template.
       #
       # @since 0.3.0
-      class MissingTemplateLayoutError < ::StandardError
+      class MissingTemplateLayoutError < Lotus::View::Error
```

Comment:
Just a thought: is there something like a search path in lotus or what extensions were searched? Then this would be useful information in the exception message.